src/avatar.tsx

```diff
@@ -86,7 +90,13 @@ const Avatar = ({
                 className={classes.avatar}
                 role="img"
                 aria-label={ariaLabel ?? initials}
-                style={{width: size, height: size, color: textColor, background: backgroundColor}}
+                style={{
+                    width: size,
+                    height: size,
+                    color: textColor,
+                    background: backgroundColor,
+                    boxShadow: border,
```

Comment:
why box `shadow` instead of `border`?

Reply:
hmmm... please, use border, I don't see a good reason for not using it

Reply:
If using border we need to decide the box-sizing of content-box or border-box to not influence on the size of component. I sugest to use box-sizing: border-box; to not make the component increse its size from 64 to 65 when applying the border.

We will be doing this change right now.